docs/_static/mxnet-theme/navbar.html

```diff
@@ -22,6 +24,7 @@ <h1 id="logo-wrap">
             <li><a class="main-nav-link" href="{{url_root}}api/c++/index.html">C++</a></li>
             <li><a class="main-nav-link" href="{{url_root}}api/clojure/index.html">Clojure</a></li>
             <li><a class="main-nav-link" href="{{url_root}}api/julia/index.html">Julia</a></li>
+            <li><a class="main-nav-link" href="https://github.com/awslabs/keras-apache-mxnet">Keras</a></li>
```

Comment:
keras is not a language binding. It's a community project and should reside somewhere else.